src/components/events-carrosel/index.js

```diff
@@ -17,13 +17,21 @@ const Events = () => {
       const events = data.map((data) => {
         if (data) {
           return {
-            nome: data.attributes.nome,
+            id: data.id,
+            name: data.attributes.nome,
             date: new Date(data.attributes.data),
-            imagem_url: data.attributes.foto_divulgacao.data.attributes.url
+            image_url: data.attributes.foto_divulgacao.data.attributes.url,
+            time_start: data.attributes.horario_inicio,
+            time_end: data.attributes.horario_fim,
```

Comment:
Já que temos a oportunidade de remapear os dados vindos do CMS, eu sugeriria aproveitar para manter a convenção do Javascript de camel case:

```suggestion
            imageUrl: data.attributes.foto_divulgacao.data.attributes.url,
            startTime: data.attributes.horario_inicio,
            endTime: data.attributes.horario_fim,
```

Daria até para organizar os dados de forma lógica também se quisermos, mas acho que a primeira opção seria mais do que suficiente para este caso:

```suggestion
            imageUrl: data.attributes.foto_divulgacao.data.attributes.url,
            duration: {
              start: data.attributes.horario_inicio
              end: data.attributes.horario_fim
            }
```